t/t2801-top-cmd.t

```diff
@@ -79,11 +79,15 @@ test_expect_success 'submit batch script and wait for it to start' '
 	cat >batch.sh <<-EOT &&
 	#!/bin/sh
 	touch job2-has-started
-	flux mini run sleep 300
+	jobid=\$(flux mini submit --flags waitable sleep 300)
+	flux job wait-event \$jobid start
```

Comment:
The separate `submit` and `wait-event` could be replaced with `flux mini submit --wait-event=start --flags waitable sleep 300`